documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Improved migration plan in the console
+
+The user interface console has improved the process of creating a migration plan.
+
+.`instance type` and `instance type preferences` have replaced {ocp-name} templates
+
+The `instance type preferences` have replaced {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+Customer Virtual Machine Mapping are custom templates that you can configure for specific migrations, VMs, and users.
+
+.Full support for migration from OVA
+
+{project-short} supports migration from OVA.
+
+.Integration of the `Create` and `Edit` migration plans
+
+The user interface `Create` and `Edit` elements for migration plans now align with the look and feel of the {ocp} web console and displays up-to-date data. These elements now replace Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
+
+.VM posted with `Running` state
+
+{project-short} shows each VM's migration `Running` state during the migration. This single `Ronning` operation replaces two previous operations, `Create/Post VM` and Run VM`.  link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+.Can download `must-gather` logs only using the CLI
+
+The {product-short} web console can no longer download logs. Every `must-gather` log image download must be preformed using CLI commands. See link:https://github.com/openshift/must-gather-operator[Must Gather Operator] for more information.
+
+.Review and configure migrated VMs
+
+The {project-short} web console shows the name and migration status of each VM. The migrated VMs are labeled `app: <vmname>` to help further changes in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
```

Comment:
```suggestion
.Migrated VMs can now be reviewed and configured

The {project-short} web console now displays the name and migration status of each virtual machine (VM). The migrated VMs have the the `app: <vmname>` lable to help further changes in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
```
Attempted to rewrite the title and description a bit. Not sure it it makes sense from the technical POV, though... As for the title, IIUC, it's now possible to somehow review and configure the migrated VMs, but the description doesn't provide details about reviewing or configuring, making the title and description kind of unlinked.... Is there, for example, some doc that describes these processes in more details? It would be great to provide a link, is such exists. Or, alternatively, add some minor details to the description so that the user understands what we're talking about.